components/avPublisher/avPublisher.c

```diff
@@ -232,7 +224,6 @@ static void HandleAvPushComplete
     LE_FATAL("Unexpected push result status %d (%s).", status, sensorPtr->obsPath);
 }
 
-
```

Comment:
This is also debatable.

Reply:
We already have giant headers on every function, so I don't think two blank lines between functions is necessary to visually break them up.  If we didn't have this already, then I think blank line makes sense.

```
//--------------------------------------------------------------------------------------------------
/**
 *
 */
//--------------------------------------------------------------------------------------------------
```